samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/BotController.cs

```diff
@@ -34,14 +35,15 @@ public class BotController : ControllerBase
     /// <summary>
     /// The constructor of BotController.
     /// </summary>
-    /// <param name="memoryStore">The memory store (optional).</param>
+    /// <param name="optionalIMemoryStore">Optional memory store.</param>
```

Comment:
nit: we should also detail when/why this optional memory store will be used in this controller. Maybe just copy and paste the comments in `SemanticKernelExtensions.cs` to here?

Reply:
Just to make things more obvious because it's little confusing.